src/View/Cell.php

```diff
@@ -259,15 +260,17 @@ protected function _cacheConfig($action)
      * This is because PHP will not allow a __toString() method to throw an exception.
      *
      * @return string Rendered cell
+     * @throws \Error Include error details for PHP 7 fatal errors.
      */
     public function __toString()
     {
         try {
             return $this->render();
         } catch (Exception $e) {
             trigger_error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()), E_USER_WARNING);
-
             return '';
+        } catch (Error $e) {
+            throw new Error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()));
```

Comment:
@thinkingmedia OK. I agree with you. Strictly speaking, you changes the error class from ParseError or something to Error. But it wouldn't cause any problems, as no one catches the Error object unless __toString() is called explicitly.

Another approach might be to call exception handler and die:
```php
// In Cell
public function __toString() {
    try {
        return (string)$this->_toString();
    } catch (Exception $e) {
        // To be honest, it would be better to not suppress any exceptions here...
    } catch (Error $e) {
        // It is difficult to write clean code to support both PHP5 and PHP7...
    }

    $handler = set_exception_handler(null);
    call_user_func($handler, $e);
    die;
}
```
This way would allow to keep the original message and stacktrace.

Regardless of how we fix this issue, I think that creating ToStringTrait might be not a bad idea, as it is not a specific issue to Cell. Every __toString() call would cause this problem if we are using PHP7. 
```php
trait ToStringTrait
{
    abstract protected function _toString();

    public function __toString() {
        try {
            return (string)$this->_toString();
        } catch (Exception $e) {} catch (Error $e) {}

        $handler = set_exception_handler(null);
        call_user_func($handler, $e);
        die;
    }
}
```